.gitignore

```diff
@@ -8,3 +8,4 @@ pkg/*
 bin/*
 files/**/cache/
 vendor/cookbooks
+files/chef-marketplace-gem/Gemfile.lock
```

Comment:
Everything should go in the project Gemfile.  We used to have a specific Gemfile for just the gem but it made doing unit testing really hard.